doc/tutorials/content/sources/qt_visualizer/CMakeLists.txt

```diff
@@ -11,7 +11,13 @@ set(CMAKE_AUTOUIC ON) # UI files
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
 
 # Find the QtWidgets library
-find_package(Qt5 REQUIRED Widgets)
+find_package(Qt6 QUIET COMPONENTS Concurrent OpenGL Widgets)
+if (NOT Qt6_FOUND)
+  find_package(Qt5 COMPONENTS Concurrent OpenGL Widgets REQUIRED)
+  set(QT_VERSION_MAJOR 5)
+else()
+  set(QT_VERSION_MAJOR 6)
+endif()
```

Comment:
I wonder if this can be in a common place... knowing cmake, maybe not. Thoughts?